### PR TITLE
Parameterize cassandra replication factor

### PR DESCRIPTION
**Pull request by rajusem**

### CODE REVIEW

File: openshift/template.yaml [hunk 1]
```diff
@@ -43,6 +43,8 @@ objects:
             value: ${REST_VALUE}
           - name: CASSANDRA_SERVICE
             value: ${CASSANDRA_SERVICE}
+          - name: DB_REPLICATION_FACTOR
```
Comment (arajkumar) [inline openshift/template.yaml]: Keep uniform prefix while naming parameters. Probably `CASSANDRA_REPLICATION_FACTOR` is better.

File: openshift/template.yaml [hunk 2]
```diff
@@ -151,4 +153,9 @@ parameters:
   displayName: Query administration region
   required: false
   name: QUERY_ADMINISTRATION_REGION
-  value: ""
\ No newline at end of file
+  value: ""
+- description: Data replication factor at database level. 
+  displayName: Database Replication Factor
+  required: true
+  name: DB_REPLICATION_FACTOR
```
Comment (arajkumar) [inline openshift/template.yaml]: ditto

File: openshift/template.yaml [hunk 3]
```diff
@@ -151,4 +153,9 @@ parameters:
   displayName: Query administration region
   required: false
   name: QUERY_ADMINISTRATION_REGION
-  value: ""
\ No newline at end of file
+  value: ""
+- description: Data replication factor at database level. 
+  displayName: Database Replication Factor
+  required: true
```
Comment (arajkumar) [inline openshift/template.yaml]: if default value is provided then `required` shall be removed.

**Review by arajkumar — CHANGES_REQUESTED** [review]

File: openshift/template.yaml (hunk 1 above)
Comment (rajusem) [reply openshift/template.yaml]: changed it to CASSANDRA_REPLICATION_FACTOR

File: openshift/template.yaml (hunk 2 above)
Comment (rajusem) [reply openshift/template.yaml]: changed it to CASSANDRA_REPLICATION_FACTOR

File: openshift/template.yaml (hunk 3 above)
Comment (rajusem) [reply openshift/template.yaml]: kept required to keep it consistent. Changed its value to `false.`

**Review by arajkumar — APPROVED** [review]

**Pull request merged**